pkg/systemd/terminal.jsx

```diff
@@ -186,6 +263,30 @@ const _ = cockpit.gettext;
                             </ToolbarContent>
                         </Toolbar>
                     </div>
+                <div className={"warn"}>
+                    {this.state.pathError && <Alert isInline variant='danger'
+                        title={'Error opening directory: ' + this.state.pathError + ' does not exist' }
```

Comment:
This needs to be i18n'ed, using `_("Error ...")`, and `cockpit.format` with a `$0` placeholder.